docs/modules/ROOT/pages/how-to-guides/releasing-applications/proc_releasing_to_app_int_services.adoc

```diff
@@ -0,0 +1,276 @@
+
+=== Verify through a Change
+
+If you don't have CLI access or don't wish to make a manual release, you can simply trigger a change in a component which passes all tests and will cause a release to be generated. We recommend you accomplish this by making a no-op change to your Dockerfile/Containerfile.
+
+=== Verify through Manual Release
+
+If you wish to verify your ReleasePlan and ReleasePlanAdmission via a manual release, you first need to choose a snapshot to release.  
+
+With CLI access to the Konflux cluster, run `oc get snapshots --sort-by=.metadata.creationTimestamp` to get snapshots sorted by age.  Select your snapshot, and use it as the snapshot value in your Release object!  
+
+You'll want to create a `release.yaml` file locally and `oc apply -f` that file.  In our example, releasing the snapshot `example-application-2fr9g` will look like:
+
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Release
+metadata:
+  name: example-application-manual-release
+  namespace: example-tenant
+spec:
+  releasePlan: example-application-releaseplan
+  snapshot: example-application-2fr9g
+----
+
+=== Watching your Release
+
+You can watch a created release under the `Releases` tab in the UI or by running `oc get releases` in the CLI.  If this Release fails, it may be a configuration error, we recommend contacting link:https://redhat.enterprise.slack.com/archives/C031USXS2FJ[#forum-konflux-release] for support with failed releases!
+
+=== Getting access to your released container
+
+Once the release succeeds, the component container(s) should be published to the repository defined in your ReleasePlanAdmission, in our case`quay.io/redhat-services-prod/example-tenant/example-component-1`.  
+
+The ReleaseEngineering and Konflux Release teams can help you get access to your repositories in link:https://redhat.enterprise.slack.com/archives/C031USXS2FJ[#forum-konflux-release], however the App-SRE service account leveraged by App-Interface will have access without intervention.  
+
+== Update your app.yaml
+
+Now that your Application's Components are released to the `redhat-services-prod` registry, you can leverage them in your App in App-Interface. 
+
+In your `app.yaml`, where you would typically include authentication to the app-sre quay repository, instead include the following:
+
+[source,yaml]
+----
+authentication:
+  image:
+    path: app-sre/quay/redhat-services-prod-pull
+    field: all
+----
+
+This will provide your application on the service cluster access to your container image(s) published to `redhat-services-prod`.  
+
+NOTE: If you are migrating from App-Interface Build to Konflux for builds, you should verify your image in `redhat-services-prod` before making the authentication and image reference change in a single MR.  
```

Comment:
following the work in [APPSRE-9856](https://issues.redhat.com/browse/APPSRE-9856), there are no app-interface under-the-hood changes needed for service owners migrating to konflux to switch from [quay.io/app-sre](http://quay.io/app-sre) / [quay.io/cloudservices](http://quay.io/cloudservices) to [quay.io/redhat-services-prod](http://quay.io/redhat-services-prod) :tada:
the tl;dr is -
one you have built images in [quay.io/redhat-services-prod](http://quay.io/redhat-services-prod):
hook them up for [continuous delivery](https://redhat-internal.slack.com/archives/CCRND57FW/p1707811838157389?thread_ts=1707811733.288799&cid=CCRND57FW) (upstream --> image)
switch to using them in your running service using an IMAGE (or similar) OpenShift template parameter

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/97922

Reply:
This is awesome - cross-linked in the documentation!